scripts/deploy

```diff
@@ -36,8 +33,9 @@ declare_kubeconfig
 import_image quay.io/submariner/lighthouse-agent
 import_image quay.io/submariner/lighthouse-coredns
 
-run_subm_clusters update_coredns_configmap
-run_subm_clusters update_coredns_deployment
 #run_subm_clusters install_service_export
 
 ${SCRIPTS_DIR}/deploy.sh "$@"
+run_subm_clusters create_ligthhouse_coredns_deployment
```

Comment:
Won't this fail for operator coz it already installs it? We can check for deploytool and only do these for helm.

Reply:
Add the check to see if config-map is already modified.